README.md

```diff
@@ -1,2 +1,60 @@
-# Read-Me-Generator
-A Node Application that generates a readme file
+#  Read me Generator
+
+## Description 
+A readme generator that generates Read me files in the fastest and easiest way possible.
+
+## Table of Contents
+
+* [Technologies](#technologies)
+* [Installation](#installation)
+* [Feature](#feature)
+* [Usage](#usage)
+* [License](#license)
+* [Questions](#questions)
+* [Screens](#screens)
+
+
+## Technologies 
+
+* JavaScript ES6
+* Node.js
+* npm
+* Inquirer
+
+## Installation 
+
+Following are the steps of installation
+
+```
+npm i
+```
+
+## Features
```

Comment:
Consider moving this section a little further up to give users a good intro into the app as soon as they open your repo :-)